v3/src/models/data/formula-manager.ts

```diff
@@ -22,11 +22,17 @@ type IFormulaMetadata = {
   registeredDisplay: string
   attributeId: string
   dataSetId: string
+  active: boolean
```

Comment:
The meaning of this flag is not entirely clear. It seems that it's false when initially registered but then made true in the `registerAllFormulas` reaction, which suggests that it's just tracking whether it's fully initialized. Perhaps a name change (isInitialized?) and/or a comment would help.

Reply:
Thanks, I agree it's a much better name. I just renamed active -> isInitialized in the last commit.